common/util/format.ts

```diff
@@ -26,6 +26,21 @@ export function formatMoney(amount: number) {
   return ENV_CONFIG.moneyMoniker + formatter.format(newAmount).replace('$', '')
 }
 
+export function formatMoneyNumber(amount: number) {
```

Comment:
There should be a way not to duplicate all the logic from the above function here.

The above function could just call this one.